pageserver/src/tenant/tasks.rs

```diff
@@ -210,7 +223,10 @@ async fn compaction_loop(tenant: Arc<Tenant>, cancel: CancellationToken) {
 /// GC task's main loop
 ///
 async fn gc_loop(tenant: Arc<Tenant>, cancel: CancellationToken) {
-    let wait_duration = Duration::from_secs(2);
+    const MAX_BACKOFF_SECS: f64 = 300.0;
```

Comment:
Need to check this compared to our current sleep amount.

Reply:
Ok, now I remembered the confusing `wait_duration`. This is looking good. We will continue to use the period for successful, but only change the error path.